manifests/startup.pp

```diff
@@ -43,6 +45,9 @@
   }
   # provided by camp2camp/systemd
```

Comment:
Unrelated, but technically puppet/systemd now that it's migrated to Vox Pupuli

Reply:
I delete it